File: projects/engineering.purdue.edu/kak/scriptingwithobjects/swocode/chap17/SOAPClientUsingWSDL.pl
#!/usr/bin/perl -w

### SOAPClientUsingWSDL.pl

use SOAP::Lite;                                                      #(A)

my $stub = SOAP::Lite->service( "file:ACMEWebService.wsdl");         #(B)
my $result = $stub->hello();                                         #(C)
print "$result\n";                                                   #(D)

File: projects/demos/engineering.purdue.edu/scriptingwithobjects/swocode/chap17/SOAPClientUsingWSDL.py
#!/usr/bin/env python

### SOAPClientUsingWSDL.py

import SOAPpy                                                        #(A)

proxy_server = SOAPpy.WSDL.Proxy( 'ACMEWebService.wsdl' )            #(B)
result = proxy_server.hello()                                        #(C)
print result                                                         #(D)
